README.md

```diff
@@ -8,35 +8,467 @@ The Pinecone Java client lets JVM applications interact with Pinecone services.
 
 Maven:
 ```
 <dependency>
   <groupId>io.pinecone</groupId>
   <artifactId>pinecone-client</artifactId>
-  <version>0.8.0</version>
+  <version>1.0.0-rc.4</version>
 </dependency>
 ```
 
 [comment]: <> (^ [pc:VERSION_LATEST_RELEASE])
 
 Gradle:
 ```
-implementation "io.pinecone:pinecone-client:0.8.0"
+implementation "io.pinecone:pinecone-client:1.0.0-rc.4"
 ```
 
 [comment]: <> (^ [pc:VERSION_LATEST_RELEASE])
 
-Alternatively, you can use our standalone uberjar [pinecone-client-0.8.0-all.jar](https://repo1.maven.org/maven2/io/pinecone/pinecone-client/0.8.0/pinecone-client-0.8.0-all.jar), which bundles the pinecone client and all dependencies together inside a single jar. You can include this on your classpath like any 3rd party JAR without having to obtain the *pinecone-client* dependencies separately.
+Alternatively, you can use our standalone uberjar [pinecone-client-1.0.0-rc.4-all.jar](https://repo1.maven.org/maven2/io/pinecone/pinecone-client/1.0.0-rc.4/pinecone-client-1.0.0-rc.4-all.jar), which bundles the pinecone 
+client and all dependencies together. You can include this in your classpath like you do with any 3rd party JAR without 
+having to obtain the *pinecone-client* dependencies separately.
 
 [comment]: <> (^ [pc:VERSION_LATEST_RELEASE])
 
+## Usage
 
-## Examples
+### Initializing the client
 
-- The data and control plane operation examples can be found in `io/pinecone/integration` folder.
+Before you can use the Pinecone Java SDK, you must sign up for a Pinecone account and find your API key in the Pinecone 
+console 
+dashboard at [https://app.pinecone.io](https://app.pinecone.io).
+
+#### Using apiKey
+
+The `Pinecone` class is your main entry point into the Pinecone Java SDK. You can instantiate the client with 
+your `apiKey`, either by passing it as an argument in your code or by setting it as an environment variable called 
+`PINECONE_API_KEY`.
+
+Note: for pod-based indexes, you will also need an `environment` variable. You can set pass this as an argument in 
+your code or set it as an environment variable called `PINECONE_ENVIRONMENT`.
+
+```java
+import io.pinecone.clients.Pinecone;
+import org.openapitools.client.model.*;
+
+public class InitializeClientExample {
+    public static void main(String[] args) {
+        Pinecone pinecone = new Pinecone.Builder("PINECONE_API_KEY").build();
+    }
+}
+```
+
+#### Passing OkHttpClient
+```java
+import io.pinecone.clients.Pinecone;
+
+public class InitializeClientExample {
+    public static void main(String[] args) {
+        OkHttpClient.Builder builder = new OkHttpClient.Builder()
+                .connectTimeout(10, java.util.concurrent.TimeUnit.SECONDS)
+                .readTimeout(30, java.util.concurrent.TimeUnit.SECONDS)
+                .writeTimeout(30, java.util.concurrent.TimeUnit.SECONDS);
+
+        OkHttpClient httpClient = builder.build();
+
+        Pinecone pinecone = new Pinecone.Builder("PINECONE_API_KEY").build();
+    }
+}
```

Comment:
For sake of brevity, I think we can exclude the Java main wrapper and import statements from all but the first of these examples.

The exception for imports would be if we're importing some various Pinecone classes besides the Pinecone client, but we can still exclude the java main wrapper in those cases.

E.g.: 
```java      
Pinecone pinecone = new Pinecone.Builder("PINECONE_API_KEY").build();
        
String indexName = "example-index";
String similarityMetric = "cosine";
Integer dimension = 1538;
String cloud = "aws";
String region = "us-west-2";

pinecone.createServerlessIndex(indexName, similarityMetric, dimension, cloud, region);
```

and 

```java
import io.pinecone.clients.Index;
import io.pinecone.clients.Pinecone;
...

Pinecone pinecone = new Pinecone.Builder("PINECONE_API_KEY").build();

Index index = pinecone.getIndexConnection("example-index");
DescribeIndexStatsResponse indexStatsResponse = index.describeIndexStats();
```

Reply:
Done!